[2025-03-13T21:22:04.517+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-13T21:22:04.529+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.transform manual__2025-03-13T21:22:03.565808+00:00 [queued]>
[2025-03-13T21:22:04.538+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.transform manual__2025-03-13T21:22:03.565808+00:00 [queued]>
[2025-03-13T21:22:04.539+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-13T21:22:04.963+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): transform> on 2025-03-13 21:22:03.565808+00:00
[2025-03-13T21:22:04.969+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=118) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-13T21:22:04.971+0000] {standard_task_runner.py:72} INFO - Started process 122 to run task
[2025-03-13T21:22:04.971+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'transform', 'manual__2025-03-13T21:22:03.565808+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpo8l758ds']
[2025-03-13T21:22:04.973+0000] {standard_task_runner.py:105} INFO - Job 24: Subtask transform
[2025-03-13T21:22:05.261+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.transform manual__2025-03-13T21:22:03.565808+00:00 [running]> on host 7499a3ac39c6
[2025-03-13T21:22:05.466+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='your_email@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T21:22:03.565808+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-13T21:22:03.565808+00:00'
[2025-03-13T21:22:05.470+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-13T21:22:05.528+0000] {logging_mixin.py:190} INFO - Starting transformation process...
[2025-03-13T21:22:05.528+0000] {logging_mixin.py:190} INFO - Data trasnformed
[2025-03-13T21:22:05.528+0000] {logging_mixin.py:190} INFO - Transformation complete.
[2025-03-13T21:22:05.535+0000] {python.py:240} INFO - Done. Returned value was: Transformation completed successfully!
[2025-03-13T21:22:05.543+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-13T21:22:05.544+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=transform, run_id=manual__2025-03-13T21:22:03.565808+00:00, execution_date=20250313T212203, start_date=20250313T212204, end_date=20250313T212205
[2025-03-13T21:22:05.563+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-13T21:22:05.577+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-13T21:22:05.578+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
